osu.Game.Rulesets.Osu/Objects/Spinner.cs

```diff
@@ -26,7 +26,10 @@ public double EndTime
         /// </summary>
         public int SpinsRequired { get; protected set; } = 1;
 
-        public int MaximumBonusSpins => SpinsRequired;
+        /// <summary>
+        /// Number of spins available to give bonus, beyond <see cref="SpinsRequired"/>.
+        /// </summary>
+        public int MaximumBonusSpins => (int)(SpinsRequired * 1.8f); // roughly matches stable
```

Comment:
This is only true at OD10. It's 2.5x less than maximum for stable at OD0.

Reply:
I think the true value is `7.95 / ({post-od value} * 0.6)`, which could be rounded up (just don't want to under-estimate stable).

Reply:
Are we actually looking to match stable 100%? stable's spin calculations are absolutely impossible to understand as a mapper or player IMO.

Reply:
I'm only concerned about under-estimating. As it is, as long as you spin faster than 200RPM you'll run out of bonus ticks at OD0. At OD5 it's 324.

Reply:
So you think lower OD should have more bonus spins? Rather than being an understandable ratio from base spins?

Reply:
If you wanted to go down that route you'd have to convince me that OD0 and OD10 should have the same amount of maximum number of spins. So long as you can't simplify that, I see no reason to simplify the number of bonus spins as if people are going to care about that and not the minimum RPM they have to spin to finish the spinner in the first place.

Reply:
I think it's actually quite understandable anyway.

```
// 2-part linear curve between OD0 and OD10
MinimumRPS = 0.6 * { 3, 5, 7.5 }

RequiredSpins = RPS * Duration

// Capped at 480RPM
MaximumRPS = 8
BonusSpins = MaximumRPS / RPS
```

Reply:
So as OD drops, the total score attainable for a map will increase beyond 1mil?

Reply:
Sure. Is that what you're trying to simplify?

Reply:
I'm just thinking we should reconsider spinning as we are with most other elements, because that seems really weird to me (easy mod score will be higher than no-mod due to bonus spin count). Maybe it makes sense? I'm not sure. We can go with it for now if you think that's best.

Reply:
That's impossible, because EZ halves your multiplier anyway. Meaning you'd need to spin ~450x more to get a higher score than a nomod player just through bonus points, which just isn't possible via the multipliers here.

Unless you're thinking mod multiplier shouldn't be applied (as we once discussed), at which point separate leaderboards or a completely different leaderboard system altogether needs to be envisioned. I don't really want to get into all that in this thread though.

How about from a player's perspective - most new players will flail their mouse around spinning, and I can't get any lower than 240-280RPM (lazer) by doing that. Considering most newbie maps are around OD2-OD4, you're looking at around max 305RPM to get all bonus points. So once a new player does anything different they're going to very quickly cap out and have nothing to work towards there.

Reply:
Yeah, thinking no-multipliers. Not that we are going to ever compare 1:1, but still feels a bit weird. Let's go with matching for now and we can re-discuss at a later point with more consideration.

Reply:
I've implemented and tested the code your specified, but it feels further off stable than what I had. Can you take a look? Do we need to increase the "max spin" rate?